test/basic/vpc_create/reachability_ops.tf

```diff
@@ -0,0 +1,129 @@
+data "aws_ami" "ubuntu" {
+  most_recent = true
+
+  filter {
+    name   = "name"
+    values = ["ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-*"]
+  }
+
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+
+  owners = ["099720109477"] # Canonical
+}
+
+data "aws_vpc" "test_vpc" {
+  depends_on = [aptible_aws_vpc.network]
+
+  tags = {
+    Name = var.vpc_name
+  }
+}
+
+data "aws_subnets" "private_subnets" {
+  filter {
+    name   = "vpc-id"
+    values = [data.aws_vpc.test_vpc.id]
+  }
+
+  tags = {
+    Network = "Private"
+  }
+}
+
+data "aws_subnets" "public_subnets" {
+  filter {
+    name   = "vpc-id"
+    values = [data.aws_vpc.test_vpc.id]
+  }
+
+  tags = {
+    Network = "Public"
+  }
+}
+
+locals {
+  private_subnet_id_to_use_for_instance = data.aws_subnets.private_subnets.ids[0]
+  public_subnet_id_to_use_for_instance  = data.aws_subnets.public_subnets.ids[0]
+}
+
+resource "aws_security_group" "allow_access_to_igw" {
```

Comment:
you can probably rename this now to be less misleading. like "open access" or something